config/passport-setup.js

```diff
@@ -0,0 +1,77 @@
+const passport = require('passport');
+const GooglePlusTokenStrategy = require('passport-google-plus-token');
+const FacebookTokenStrategy = require('passport-facebook-token');
+const keys = require('./keys');
+const { Add_New_User } = require('../src/queries/Add_New_User');
+const { Find_User } = require('../src/queries/Find_User');
+
+// Google OAuth Strategy
+passport.use('googleToken', new GooglePlusTokenStrategy({
+    clientID: keys.google.clientID,
+    clientSecret: keys.google.clientSecret
+}, async (accessToken, refreshToken, profile, done) => {
+    try {
+        // Should have full user profile over here
+        console.log('profile', profile);
+        console.log('accessToken', accessToken);
+        console.log('refreshToken', refreshToken);
+
+        console.log('givenName', profile.name.givenName);
+
+        //check if user already exists in our DB
+        console.log('profileId******', profile.id)
```

Comment:
we don;t need these console logs here

Reply:
there is not passport-setup anymore i removed it because we decided do not use in passport